tests/run-with-compiler/scala2-library-from-tasty-jar.scala

```diff
@@ -0,0 +1,74 @@
+import dotty.tools.io.Directory
+import dotty.tools.dotc.util.ClasspathFromClassloader
+
+import java.io.File.pathSeparator
+import java.io.File.separator
+
+@main def Test: Unit =
+  blacklistsOnlyContainsClassesThatExist()
+  // FIXME this test does not work on JDK8
+  // Caused by: dotty.tools.dotc.core.TypeError$$anon$1: package scala.quoted.runtime.Expr does not have a member method quote
+  if System.getProperty("java.specification.version") != "1.8" then
```

Comment:
This is a possible bug when running the from-TASTy compiler on JDK8 or with the test itself. I will open an issue to investigate this further.

Note that the Windows tests found a bug in `TASTYRun.scala` that was fixed but then failed due it running on JDK8. Therefore this fix will not be tested until we fix this issue or run Windows tests on a newer version of the JDK.